corehq/apps/auditcare/tests/test_tasks.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime
+
+from ..models import AccessAudit, NavigationEventAudit
+from ..tasks import copy_events_to_sql
+from .testutils import AuditcareTest, delete_couch_docs, save_couch_doc
+
+
+class TestCopyEventsToSQL(AuditcareTest):
+    username = "audit@care.com"
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        username = "couch@test.com"
+        cls.couch_ids = [
+            save_couch_doc(
+                "NavigationEventAudit",
+                username,
+                event_date=datetime(2021, 2, 1, 2).strftime("%Y-%m-%dT%H:%M:%SZ"),
+                description='User Name',
+                extra={},
+                headers={
+                    'REQUEST_METHOD': 'GET',
+                },
+                ip_address='10.1.2.3',
+                request_path='/a/delmar/phone/restore/?version=2.0&since=...',
+                session_key='abc123',
+                status_code=200,
+                user_agent='okhttp/4.4.1',
+                view_kwargs={'domain': 'delmar'},
+                view='corehq.apps.ota.views.restore',
+            ),
+            save_couch_doc(
+                "AccessAudit",
+                username,
+                event_date=datetime(2021, 2, 1, 3).strftime("%Y-%m-%dT%H:%M:%SZ"),
+                access_type='login',
+                description='Login Success',
+                failures_since_start=None,
+                get_data=[],
+                http_accept='text/html',
+                ip_address='10.1.3.2',
+                path_info='/a/delmar/login/',
+                post_data=[],
+                session_key='abc123',
+                user_agent='Mozilla/5.0',
+            ),
+            save_couch_doc(
+                "NavigationEventAudit",
+                username,
+                event_date=datetime(2021, 2, 1, 5).strftime("%Y-%m-%dT%H:%M:%SZ"),
+                description='User Name',
+                extra={},
+                headers={
+                    'REQUEST_METHOD': 'GET',
+                },
+                ip_address='10.1.2.3',
+                request_path='/a/sandwich/phone/restore/?version=2.0&since=...',
+                session_key='abc123',
+                status_code=200,
+                user_agent='okhttp/4.4.1',
+                view_kwargs={'domain': 'sandwich'},
+                view='corehq.apps.ota.views.restore',
+            ),
+        ]
+
+    @classmethod
+    def tearDownClass(cls):
+        delete_couch_docs(cls.couch_ids)
+        super().tearDownClass()
+
+    def tearDown(self):
+        AccessAudit.objects.all().delete()
+        NavigationEventAudit.objects.all().delete()
+        super().tearDown()
```

Comment:
This is not necessary since each test is done in a separate transaction, so SQL data is automatically cleaned up.

Reply:
29099450e61b71bb98abae04a254b34c89670b8c